docs-2.0/3.ngql-guide/8.clauses-and-options/where.md

```diff
@@ -161,11 +161,15 @@ nebula> MATCH (v:player) \
 nebula> MATCH (v:player) \
         WHERE exists(v.age) \
         RETURN v.name, v.age;
-
++-------------------------+-------+
+| v.name                  | v.age |
++-------------------------+-------+
+| "Boris Diaw"            | 36    |
++-------------------------+-------+
+| "DeAndre Jordan"        | 30    |
++-------------------------+-------+
 ```
```

Comment:
Could you check the output of this query? I believe all players have the property age.

Reply:
I deleted most rows since too long.